guide/C/ch_invest.xml

```diff
@@ -1903,54 +1932,259 @@ Income
+                <entry>
+                  <emphasis>Sell</emphasis>
+                </entry>
+              </row>
+
+              <row>
+                <entry>
+                  Assets:Bank ABC
+                </entry>
+
+                <entry></entry>
+
+                <entry></entry>
+
+                <entry>
+                  3525.00
+                </entry>
+
+                <entry></entry>
+              </row>
+
+              <row>
+                <entry>
+                  Assets:Brokerage Account:Stock:AMZN
+                </entry>
+
+                <entry>
+                  0
+                </entry>
+
+                <entry>
+                  0
+                </entry>
+
+                <entry>
+                  1600.00
+                </entry>
+
+                <entry></entry>
+              </row>
+
+              <row>
+                <entry>
+                  Expenses:Commission
+                </entry>
+
+                <entry></entry>
+
+                <entry></entry>
+
+                <entry>
+                  75.00
+                </entry>
+
+                <entry></entry>
+              </row>
+
+              <row>
+                <entry>
+                  Assets:Brokerage Account:Stock:AMZN
+                </entry>
+
+                <entry>
+                  &minus;100
+                </entry>
+
+                <entry>
+                  36.00
+                </entry>
+
+                <entry></entry>
+
+                <entry>
+                  3600.00
+                </entry>
+              </row>
+
+              <row>
+                <entry>
+                  Income:Capital Gain (Long Term):AMZN
+                </entry>
+
+                <entry></entry>
+
+                <entry></entry>
+
+                <entry></entry>
+
+                <entry>
+                  1600.00
+                </entry>
+              </row>
+            </tbody>
+          </tgroup>
+        </table>
+        </para>
+        <para>
+          <figure pgwide="1">
+          <title>Selling Shares, Gain Combined, Gross Pricing</title>
+          <screenshot id="invest-sellstockManProfComb">
+            <mediaobject>
+              <imageobject role="html">
+                <imagedata fileref="figures/invest_sellstock.png" format="PNG"
+                           srccredit="Chris Good" width="510px"></imagedata>
+              </imageobject>
+
+              <imageobject role="fo">
+                <imagedata fileref="figures/invest_sellstock.png" format="PNG"
+                           srccredit="Chris Good"></imagedata>
+              </imageobject>
+    
+              <textobject><phrase>Selling Shares,Gain Combined, Gross Pricing</phrase></textobject>
+    
+            </mediaobject>
+          </screenshot>
+          </figure>
+          <note>
+            <para>In the above screenshot, it appears there are 2 transactions for Mar 21st, 2006. This is because the
```

Comment:
Didn't we want to change `Mar 21st, 2006`?
Or add above a comment: Redo screenshot with new date format

Reply:
ab6c9581